src/TM_Tutor_Tables.c

```diff
@@ -3,15 +3,15 @@
 
 const u16 gTMHMMoves[NUM_TMSHMS] =
 {
-	MOVE_FOCUSPUNCH,	//1
+	MOVE_WORKUP,		//1
```

Comment:
some extra white spaces, not a huge deal tho

Reply:
Extra spacing is to realign the TM/HM number comments actually